packages/react-topology/src/types.ts

```diff
@@ -142,6 +142,18 @@ export interface ScaleDetailsThresholds {
   medium: number;
 }
 
+type Never<Type> = { [K in keyof Type]?: never };
+type EitherNotBoth<TypeA, TypeB> = (TypeA & Never<TypeB>) | (TypeB & Never<TypeA>);
+
+interface ViewPaddingPixels {
+  padding: number; // Set this to Number.POSITIVE_INFINITY to bypass the checks
```

Comment:
This comment `// Set this to Number.POSITIVE_INFINITY to bypass the checks` is no longer relevant correct?